### ISSUE TITLE
Running against adam built for scala version 2.11 does not work

### ISSUE THREAD

**Issue by heuermh** [heuermh/adam-plugins#1]
As might be expected, running `adam-plugins-0.16.1-SNAPSHOT.jar` against adam built for scala version 2.11 does not work

``` bash
$ ADDL_JARS=adam-plugins-0.16.1-SNAPSHOT.jar bin/adam-submit \
>         plugin com.github.heuermh.adam.plugins.CountAlignments \
>         adam-core/src/test/resources/small.sam
Exception in thread "main" java.lang.NoSuchMethodError: scala.Predef$.$conforms()Lscala/Predef$$less$colon$less;
    at org.bdgenomics.adam.cli.ADAMMain$.main(ADAMMain.scala:104)
    at org.bdgenomics.adam.cli.ADAMMain.main(ADAMMain.scala)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:606)
    at org.apache.spark.deploy.SparkSubmit$.org$apache$spark$deploy$SparkSubmit$$runMain(SparkSubmit.scala:569)
    at org.apache.spark.deploy.SparkSubmit$.doRunMain$1(SparkSubmit.scala:166)
    at org.apache.spark.deploy.SparkSubmit$.submit(SparkSubmit.scala:189)
    at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:110)
    at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala)
```

**Comment by heuermh** [comment]
Perhaps the problem isn't with adam-plugins

``` bash
$ ADDL_JARS=adam-plugins_2.11-0.16.1-SNAPSHOT.jar bin/adam-submit \
>          plugin com.github.heuermh.adam.plugins.CountAlignments \
>         adam-core/src/test/resources/small.sam
Exception in thread "main" java.lang.NoSuchMethodError: scala.Predef$.$conforms()Lscala/Predef$$less$colon$less;
    at org.bdgenomics.adam.cli.ADAMMain$.main(ADAMMain.scala:104)
    at org.bdgenomics.adam.cli.ADAMMain.main(ADAMMain.scala)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:606)
    at org.apache.spark.deploy.SparkSubmit$.org$apache$spark$deploy$SparkSubmit$$runMain(SparkSubmit.scala:569)
    at org.apache.spark.deploy.SparkSubmit$.doRunMain$1(SparkSubmit.scala:166)
    at org.apache.spark.deploy.SparkSubmit$.submit(SparkSubmit.scala:189)
    at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:110)
    at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala)

$ bin/adam-submit transform adam-core/src/test/resources/small.sam /tmp/small.adam
Exception in thread "main" java.lang.NoSuchMethodError: scala.Predef$.$conforms()Lscala/Predef$$less$colon$less;
    at org.bdgenomics.adam.cli.ADAMMain$.main(ADAMMain.scala:104)
    at org.bdgenomics.adam.cli.ADAMMain.main(ADAMMain.scala)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:606)
    at org.apache.spark.deploy.SparkSubmit$.org$apache$spark$deploy$SparkSubmit$$runMain(SparkSubmit.scala:569)
    at org.apache.spark.deploy.SparkSubmit$.doRunMain$1(SparkSubmit.scala:166)
    at org.apache.spark.deploy.SparkSubmit$.submit(SparkSubmit.scala:189)
    at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:110)
    at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala)

$ which scala
/usr/local/bin/scala

$ ls -als `which scala`
8 lrwxr-xr-x  1 user  admin  32 Mar 16 15:15 /usr/local/bin/scala -> ../Cellar/scala/2.11.6/bin/scala
```

**Comment by heuermh** [comment]
Fixed by commit 1274bff02aefb8ff2d6fbe616544ac3daacb0e10
